sql_metadb/derived_tables/item_effective_callno_components.sql

```diff
@@ -0,0 +1,34 @@
+--metadb:table item_effective_callno_components
+
+--Create item_effective_callno_components that extracts all the components from a call number.
+
+DROP TABLE IF EXISTS item_effective_callno_components;
+
+CREATE TABLE item_effective_callno_components AS
+SELECT
+    items.id AS item_id,
+    jsonb_extract_path_text(items.jsonb, 'hrid') AS item_hrid,
+    jsonb_extract_path_text(items.jsonb,'effectiveCallNumberComponents','prefix') AS item_effective_call_number_prefix,
+    jsonb_extract_path_text(items.jsonb,'effectiveCallNumberComponents','callNumber') AS item_effective_call_number,
+    jsonb_extract_path_text(items.jsonb,'effectiveCallNumberComponents','suffix') AS item_effective_call_number_suffix,
+    jsonb_extract_path_text(items.jsonb,'effectiveCallNumberComponents','typeId')::uuid AS item_effective_call_number_type_id,
+    jsonb_extract_path_text(cnt.jsonb, 'name') AS item_effective_call_number_type_name
+FROM
+    folio_inventory.item__ AS items
+    LEFT JOIN folio_inventory.call_number_type__ AS cnt ON 
```

Comment:
I wondered if this query was intended to use current tables instead of the main tables, `folio_inventory.item__` and `folio_inventory.call_number_type__`.